quinn-proto/src/congestion.rs

```diff
@@ -58,6 +58,9 @@ pub trait Controller: Send {
         lost_bytes: u64,
     );
 
+    /// The known MTU for the current network path has been updated
+    fn on_mtu_update(&mut self, current_mtu: u16);
```

Comment:
Should we call the argument `new_mtu` instead?

Reply:
Sure, that seems indeed a bit clearer